### PR TITLE
Unload tpm module to support booting ISOs with GRUB 2.04 on UEFI systems

### PR DESCRIPTION
**Pull request by mika**
See https://help.ubuntu.com/community/Grub2/ISOBoot#Menuentry_Example

Closes: #975835
Thanks: Vasek Opekar <opekar@eccam.com> for the bug report

### CODE REVIEW

**Comment by mika** [comment]
Quoting myself from https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=975835#10:

```
Though we need to verify that this indeed fixes the situation,
and also doesn't cause any problems on non-UEFI systems and
on systems with different GRUB versions.
I'd very much appreciate any testing and feedback!
```

File: 42_grml [hunk 1]
```diff
@@ -116,6 +116,7 @@ ${grub_prep}
         export iso_path
         kernelopts=" $CUSTOM_BOOTOPTIONS $additional_param "
         export kernelopts
+        rmmod tpm
```
Comment (zeha) [inline 42_grml:119]: i think that needs at least a comment why this is here.

Comment (mika) [reply 42_grml:120]: Good idea, thanks, done!

**Review by zeha — APPROVED** [review]

**Comment by mika** [comment]
Thanks for your review, @zeha! Uploaded :)

**Pull request merged**